test/clojure/core_test/number_range.cljc

```diff
@@ -0,0 +1,29 @@
+(ns clojure.core-test.number-range)
+
+(def MAX-INT #?(:clj Long/MAX_VALUE
```

Comment:
Please lower case all of these vars. Sure, they're constants, but they're just vars.

Reply:
I've adopted this. Your use of the word "constants" reminded me that this is a compiler hint that can be provided via metadata. I've added it, but am happy to remove.